src/ObjectUtil/ObjectUtil.spec.ts

```diff
@@ -103,6 +103,7 @@ describe('ObjectUtil', () => {
         obj[Object.prototype.toString.call(unexpectedKey)] = true;
       });
       unexpectedKeys.forEach(unexpectedKey => {
+        // @ts-ignore
         expect(ObjectUtil.getValue(unexpectedKey, obj)).toBeUndefined();
```

Comment:
Why do we have to `ts-ignore` this line?

Reply:
It wasn't. I removed it.